src/vs/workbench/contrib/notebook/browser/contrib/find/notebookFindReplaceWidget.ts

```diff
@@ -164,7 +164,8 @@ class NotebookFindInput extends FindInput {
 		readonly instantiationService: IInstantiationService,
 		parent: HTMLElement | null,
 		contextViewProvider: IContextViewProvider,
-		options: IFindInputOptions
+		options: IFindInputOptions,
+		filterStartVisibliity = true // TODO: create notebook find options to define this
```

Comment:
yeah maybe a `CtorOptions`

Reply:
I was originally thinking of just extending the `IFindInputOptions` to make a `IFindInputOptions` that includes this, but we can also go with a `CtorOptions` (that lives alongside the `IFindInputOptions` ) if that's how we usually do it.